### PR TITLE
jobs, jstack, kill, killall: add more information link

### PR DESCRIPTION
**Pull request by blueskyson**

### CODE REVIEW

File: pages/common/kill.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > Sends a signal to a process, usually related to stopping the process.
 > All signals except for SIGKILL and SIGSTOP can be intercepted by the process to perform a clean exit.
+> More information: <https://manned.org/kill>.
```
Comment (navarroaxel) [inline pages/common/kill.md:5]: Did you see the GNU link?
```suggestion
> More information: <https://www.gnu.org/software/coreutils/manual/html_node/kill-invocation.html>.
```

Comment (blueskyson) [reply pages/common/kill.md:5]: Yes. But I think the manned.org one is more organized and detailed.

Comment (marchersimon) [reply pages/common/kill.md:5]: Looks like the version of `kill` installed on most Linux distros isn't from GNU, but from util-linux. I don't know about OSX tough.

File: pages/common/jstack.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # jstack
 
 > Java Stack Trace Tool.
+> More information: <https://www.unix.com/man-page/mojave/1/jstack/>.
```
Comment (marchersimon) [inline pages/common/jstack.md:4]:
```suggestion
> More information: <https://manned.org/jstack>.
```

File: pages/common/jobs.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # jobs
 
 > Display status of jobs in the current session.
+> More information: <https://www.gnu.org/software/bash/manual/html_node/Job-Control-Builtins.html>.
```
Comment (marchersimon) [inline pages/common/jobs.md:4]: The `manned.org` page is far more detailed here.
```suggestion
> More information: <https://manned.org/jobs>.
```

**Review by marchersimon — APPROVED** [review]
👍🏽

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**